### PR TITLE
Bump up Cephadm Ansible collection

### PR DESCRIPTION
**Pull request by cityofships**

### CODE REVIEW

File: etc/kayobe/ansible/requirements.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 collections:
   - name: stackhpc.cephadm
-    version: 1.15.1
+    version: 1.17.0
```
Comment (markgoddard) [inline etc/kayobe/ansible/requirements.yml:4]: Any notable changes to add to a reno?

Comment (jackhodgkiss) [reply etc/kayobe/ansible/requirements.yml:4]: I would think these two would be worth a mention which make cephadm key stateless and makes reef default.

https://github.com/stackhpc/ansible-collection-cephadm/pull/145
https://github.com/stackhpc/ansible-collection-cephadm/pull/152

Comment (markgoddard) [reply etc/kayobe/ansible/requirements.yml:4]: We do override the version, so ref change won't affect us.

**Comment by markgoddard** [comment]
I've proposed bumping to 1.18.0 in https://github.com/stackhpc/stackhpc-kayobe-config/pull/1219 (draft), and added a reno.

**Comment by MoteHue** [comment]
This is good to close, now https://github.com/stackhpc/stackhpc-kayobe-config/pull/1219 is merged?

**Pull request closed without merging**